script/deploy/deploy-comptroller.js

```diff
@@ -0,0 +1,7 @@
+const contractConfig = require("./contract-config");
```

Comment:
Should definitely take these from `../network` files. The current ones are clearly obsolete, the (more or less) up-to-date ones are in the venus-config repo. They don't have rinkeby, though, as we never dployed to rinkeby